code/controllers/subsystem/tickets/SSmentor_tickets.dm

```diff
@@ -71,6 +72,22 @@ GLOBAL_REAL(SSmentor_tickets, /datum/controller/subsystem/tickets/mentor_tickets
 	if(message_key == null)
 		T.staffAssigned = null //if they cancel we dont need to hold this ticket anymore
 		return
+	if(message_key == "NCT Dispatch")
+		var/mob/living/carbon/human/trainee
+		var/mob/living/carbon/human/nct
+		trainee = get_mob_by_ckey(T.client_ckey)
+		for(var/mob/living/carbon/human/I in GLOB.player_list)
+			if(!I.mind)
+				continue
+			if(I.mind.assigned_role == "Nanotrasen Career Trainer")
+				nct = I
+				break
```

Comment:
This will only ever grab one NCT, is that the intention?

Reply:
It will pick the same NCT every time, I suggest adding all NCTs to a list and then using `pick()` to get a random NCT